libcontainer/nsenter/namespace.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014, 2015 Docker Inc.
+ * Copyright 2015, 2016 The Linux Foundation
```

Comment:
i'm also not a fan of starting to add these on the files, we have a overall license and it applies to all files in the repo, we don't need them per file